src/tests/ftest/daos_test/suite.py

```diff
@@ -434,3 +434,18 @@ def test_daos_dedup(self):
         :avocado: tags=DaosCoreTest,test_daos_dedup
         """
         self.run_subtest()
+
+    def test_daos_upgrade(self):
+        """Jira ID: DAOS-10681
+
+        Test Description:
+            Run daos_test -P
+
+        Use cases:
+            Core tests for daos_test
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium
```

Comment:
Due to recent change, this should also be tagged as provider testing
```suggestion
        :avocado: tags=hw,medium,provider
```

Reply:
ah, thanks.